sample/src/main/java/io/samagra/oce_sample/ODKFeatureTesterActivity.kt

```diff
@@ -67,6 +70,7 @@ class ODKFeatureTesterActivity : AppCompatActivity(), View.OnClickListener {
 
         ODKProvider.init(application)
         odkInteractor = ODKProvider.getOdkInteractor()
+        formsInteractor = ODKProvider.getFormsInteractor()
```

Comment:
This will crash the app due to `NullPointerException`. Please initialize this object only after ODK is finished with the setup, similar to how other objects are initialized.

Reply:
Initialized `formsInteractor` in `onProcessComplete()` function